hivemind/client/dht_ops.py

```diff
@@ -0,0 +1,67 @@
+from functools import partial
+from typing import Sequence, Dict, List, Tuple, Optional
+
+from hivemind.dht import DHT, DHTNode, DHTExpiration, DHTValue
+from hivemind.client.expert import RemoteExpert
+from hivemind.client.expert_uid import (ExpertUID, ExpertPrefix, FLAT_EXPERT, Coordinate,
+                                        UID_DELIMITER, UID_PATTERN, is_valid_uid, split_uid)
+from hivemind.utils import Endpoint, get_dht_time
+
+
+def declare_experts(dht: DHT, uids: Sequence[ExpertUID], endpoint: Endpoint,
```

Comment:
I think that expert declaration can be moved outside of hivemind.client, because this is strictly the responsibility of the Server. What do you think about keeping dht_ops in hivemind.dht for now?